Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -343,6 +343,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
           "res/actions/music.png")
       .AddCodeOnlyParameter("currentScene", "")
       .MarkAsComplex();
+  extension
+      .AddAction(
+          "FadeVolume",
+          _("fade volume of a sound."),
+          _("fade the volume an audio sound on a channel to a volume within a given time."),
```

Comment:
This sentence reads weird, but I don't speak english natively so hard to suggest something else.

Reply:
I don't speak English natively either, but I am able to at the very least compose better sentences. Here would be my suggestions.
Line 349: "Fade the volume level of an audio track"
Line 350: "Fade the volume level of a sound channel within a given time"